openpgp-dsm/src/lib.rs

```diff
@@ -672,27 +684,47 @@ pub fn generate_key(
     )
     .context("could not create primary key")?;
 
-    info!("key generation: create signing subkey");
-    let signing_subkey = PublicKey::create(
-        &dsm_client,
-        key_name.to_string(),
-        KeyRole::SigningSubkey,
-        &algorithm,
-        exportable,
-        validity_period
-    )?;
+    let mut signing_subkeys: Vec<PublicKey> = vec![];
+    let mut encryption_subkeys: Vec<PublicKey> = vec![];
+    for flag in &key_flag_args {
```

Comment:
For paranoia, do you want to explicitly skip the first argument here (since you handle the primary key later)?

Reply:
Done.

Reply:
Gracias!

Resolving then :)